tests/tests/trace_api.rs

```diff
@@ -17,7 +18,7 @@ use starknet::core::types::MaybePendingBlockWithTxHashes;
 use starknet::providers::Provider;
 
 const TRACING_BLOCK_NUMBER: u64 = 0x3;
-const TRANSACTIONS_COUNT: usize = 1;
+const TRANSACTIONS_COUNT: usize = 5;
```

Comment:
Maybe we could just put a small line of documentation for each of these lines as this is not trivial to understand where they come from.